src/backend/EdFi.DataManagementService.Backend.Postgresql/Deploy/Scripts/0001_Create_Documents_Table.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS documents (
+  id BIGINT GENERATED ALWAYS AS IDENTITY(START WITH 1 INCREMENT BY 1),
+  document_partition_key SMALLINT NOT NULL,
+  document_uuid CHAR(36) NOT NULL,
+  resource_name VARCHAR(256) NOT NULL,
+  edfi_doc BYTEA NOT NULL,
```

Comment:
In SQL Server, we found that varchar instead of varbinary gave better performance. Would it make sense to use varchar here?

Reply:
Changed to `varchar`